src/Query/Query.ts

```diff
@@ -1,22 +1,22 @@
-import { TaskLayoutOptions } from '../Layout/TaskLayoutOptions';
+import { getSettings } from '../Config/Settings';
+import type { IQuery } from '../IQuery';
 import { QueryLayoutOptions } from '../Layout/QueryLayoutOptions';
+import { TaskLayoutComponent, TaskLayoutOptions } from '../Layout/TaskLayoutOptions';
+import { errorMessageForException } from '../lib/ExceptionTools';
+import { logging } from '../lib/logging';
 import { expandPlaceholders } from '../Scripting/ExpandPlaceholders';
 import { makeQueryContext } from '../Scripting/QueryContext';
 import type { Task } from '../Task/Task';
-import type { IQuery } from '../IQuery';
-import { getSettings } from '../Config/Settings';
-import { errorMessageForException } from '../lib/ExceptionTools';
-import { logging } from '../lib/logging';
-import { Sort } from './Sort/Sort';
-import type { Sorter } from './Sort/Sorter';
-import { TaskGroups } from './Group/TaskGroups';
+import { Explainer } from './Explain/Explainer';
+import type { Filter } from './Filter/Filter';
 import * as FilterParser from './FilterParser';
 import type { Grouper } from './Group/Grouper';
-import type { Filter } from './Filter/Filter';
+import { TaskGroups } from './Group/TaskGroups';
 import { QueryResult } from './QueryResult';
 import { scan } from './Scanner';
 import { SearchInfo } from './SearchInfo';
-import { Explainer } from './Explain/Explainer';
+import { Sort } from './Sort/Sort';
+import type { Sorter } from './Sort/Sorter';
 
```

Comment:
There's nothing we can do about it, but this is again going to create conflicts for people with forks that have modifications to these files (including me!)

Once the dependencies feature is released, I would really like to address #2456 - hopefully in a way independent of WebStorm.

Reply:
Yes, this is a pain indeed. The problem here is that new imports are added and I don't know whether it's ok to take them in manually. Feel free to suggest a way, I will amend this PR.

Another option is to just accept this - once all the files will be parsed WS, there will be no changes like that I suppose.

Reply:
I think it's too late to do anything about it now.